packages/node_modules/@ciscospark/plugin-phone/test/integration/spec/call.js

```diff
@@ -186,6 +188,22 @@ describe(`plugin-phone`, function() {
               .then(() => assert.equal(call.status, `disconnected`))
           ]));
         });
+
+        it(`is not sending media`, () => {
+          call = spock.spark.phone.dial(mccoy.email);
+          assert.equal(call.status, `initiated`);
+          return handleErrorEvent(call, () => Promise.all([
+            mccoy.spark.phone.when(`call:incoming`)
+              .then(([c]) => c.reject()),
+            call.when(`disconnected`)
+              .then(() => assertLocusMediaState(call, {
+                sendingAudio: false,
```

Comment:
I'm not sure I follow on this one. Isn't that what `assertLocusMediaState` does? What would I check otherwise? `call.media.peer.something`?